components/signin-button.tsx

```
@@ -45,29 +46,52 @@ const GoogleIcon = () => {
-        <Link href="/dashboard">Go to dashboard</Link>
-      </Button>
+      <div className="group relative">
+        <Button
+          size="lg"
+          className="relative cursor-pointer overflow-hidden rounded-xl border-2 border-neutral-600 bg-gradient-to-r from-black via-neutral-400 to-black px-8 py-6 text-lg font-bold text-white transition-all duration-500 ease-out hover:scale-105 active:scale-95"
+        >
+          <Link href="/dashboard">
+            <span className="relative z-10 flex items-center gap-3 font-bold tracking-wide">
+              <LayoutDashboard className="h-5 w-5 transition-all duration-500 group-hover:scale-105" />
+              Launch Dashboard
+              <div className="ml-1 h-2 w-2 animate-pulse rounded-full bg-green-200"></div>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)